source/docs/advanced/theme/analytics.md

```diff
@@ -385,7 +412,7 @@ PixelIntegration.prototype.event = function (track) {
     img.setAttribute("src", pixelUrl);
     img.setAttribute(
       "style",
-      "position: absolute; left: -5000px; top: -4000px;"
+      "position: absolute; left: -5000px; top: -4000px;",
```

Comment:
This is a strange autoformat as we do not use this rule in FC itself. 
Was it from the a formatting config inside the project or a tool you have ?

Reply:
I think it comes from my VSCode's Prettier addon. I though it used the config found in the opened folder, but it seems it has a default config somewhere.
Do we use a specific formatter at FC (if any) ?

Reply:
@abailleulfc Suggestion: Should we be really exhaustive in detailing prettier rules in `.prettierrc`, so that the IDEs don't override them?

Reply:
Btw, I just spotted `"trailingComma": "all"` in the `.prettierrc`, so I guess we do indeed have this rule?

Reply:
Then for this PR it looks good to me (moving the topic to slack)

Reply:
Config updated in #342. I'm marking this conversation as resolved.